.ebextensions/cloudwatch_linux_metrics_report.config

```diff
@@ -0,0 +1,22 @@
+packages:
+    yum:
+        perl-Switch: []
+        perl-DateTime: []
+        perl-Sys-Syslog: []
+        perl-LWP-Protocol-https: []
+
+commands:
+  1_download_scripts_package:
+    command: "wget -O /root/cw.zip http://ec2-downloads.s3.amazonaws.com/cloudwatch-samples/CloudWatchMonitoringScripts-v1.1.0.zip"
+
+  2_unzip_package:
+    command: "unzip /root/cw.zip -d /root/"
+
+  3_cleanup:
+    command: "rm -f /root/cw.zip"
+
+  4_add_line_to_tmp:
+    command: "echo '*/1 * * * * /root/aws-scripts-mon/mon-put-instance-data.pl --mem-util --mem-used --mem-avail --swap-util --swap-used --disk-path=/ --disk-space-util --disk-space-used --disk-space-avail' >>/tmp/cloudwatchcron"
```

Comment:
Are we ok with this reporting every minute? Also what user will cron execute this as; it looks like it would be root, can it not be?